### PR TITLE
Update filedrophandler.js

### PR DESCRIPTION
**Pull request by supolik**
fix: Error accessing effectAllowed property in dragover event in IE

### CODE REVIEW

File: closure/goog/events/filedrophandler.js [hunk 1]
```diff
@@ -196,7 +196,9 @@ goog.events.FileDropHandler.prototype.onElemDragOver_ = function(e) {
     e.stopPropagation();
     // Allow the drop on the drop zone.
     var dt = e.getBrowserEvent().dataTransfer;
-    dt.effectAllowed = 'all';
```
Comment (Ubehebe) [inline closure/goog/events/filedrophandler.js]: Instead of the `try` ... `catch`, how about
```
if (dt.effectAllowed) {
  dt.effectAllowed = 'all';
}
```

**Comment by Ubehebe** [comment]
Please also write a test in filehandler_test.js that fails at HEAD but passes under this patch.

**Comment by concavelenz** [comment]
No CLA found: https://github.com/google/closure-library/blob/master/CONTRIBUTING

**Comment by concavelenz** [comment]
No response.

**Pull request closed without merging**